src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -3655,9 +3686,13 @@ const static std::unordered_map<uint32_t, GfxOpcodeHandlerFunc> s2dexHandlers =
     { G_RDPHALF_2, gfx_stubbed_command_handler },
 };
 
-const static std::array<const std::unordered_map<uint32_t, GfxOpcodeHandlerFunc>*, UcodeHandlers::ucode_max>
+const static std::array<const std::unordered_map<int8_t, GfxOpcodeHandlerFunc>*, UcodeHandlers::ucode_max>
```

Comment:
@KiritoDv why is this being changed to `int8_t` from `uint32_t`?

Reply:
response from discord

> Srry i dont have git rn so let me answer here
Its because of two reasons
One because there is no more than 255 cmds, and because f3d and f3dex uses a lot opcodes with negative values on some machines it fails to interpret them
In this case my mac
It crashes because it fails to find anything